it/index.md

```diff
@@ -1,24 +1,32 @@
 ---
 title: Elixir School
 layout: home
-version: 1.0.0
+version: 2.0.0
 ---
 
-[![License](//img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
-
-Lezioni sul linguaggio di programmazione Elixir, ispirato a [Scala School](http://twitter.github.io/scala_school/) di Twitter.
-
-_Sei incoraggiato a commentare o partecipare!_
-
-## About Elixir
-
-"Elixir è un linguaggio dinamico e funzionale, progettato per costruire applicazioni scalabili e manutenibili." — [elixir-lang.org](http://elixir-lang.org/)
-
-Elixir adotta la robusta Erlang VM (BEAM) per costruire sistemi distribuiti, tolleranti ai guasti e con bassa latenza.
-
-__Features__:
-
-+ Scalabile
-+ Tollerante ai guasti
-+ Programmazione Funzionale
-+ Estensibile
+Elixir School è la destinazione pricipale per chiunque abbia la necessità di imparare e padroneggiare il linguaggio di programmazione Elixir.
+
+Che tu sia un esperto o un programmatore alle prime armi, in queste lezioni e nelle risorse ausiliari troverai tutto ciò di qui hai bisogno.
```

Comment:
Maybe `ausiliarie` (female) rather than `ausiliari` (male or neutral). The term `risorse` (and even `lezioni`) is female.

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b